appveyor.yml

```diff
@@ -95,6 +95,8 @@ on_success:
           python setup.py sdist bdist_wheel bdist_wininst bdist_egg
           Invoke-Expression "twine upload -u $env:PYPI_USERNAME -p $env:PYPI_PASSWORD --repository-url https://upload.pypi.org/legacy/ --skip-existing dist/*" 2>&1 | Out-Null
       }
+  - "conda deactivate"
+  - "conda env remove -n %ENV_NAME%"
```

Comment:
I think this should go into `on_finish:` section so its always executed

Reply:
https://www.appveyor.com/docs/appveyor-yml/

Reply:
good point, doing now